src/commands/convert/trufflehog2hdf.ts

```diff
@@ -0,0 +1,42 @@
+import { Command, Flags } from "@oclif/core";
+import fs from "fs";
+import { TrufflehogResults as Mapper } from "@mitre/hdf-converters";
+import { checkSuffix } from "../../utils/global";
+
+export default class Trufflehog2HDF extends Command {
+  static usage =
+    "convert Trufflehog2hdf -i <Trufflehog-json> -o <hdf-scan-results-json>";
```

Comment:
```suggestion
    "convert trufflehog2hdf -i <trufflehog-json> -o <hdf-scan-results-json>";
```
Keep the command in lowercase. Apply this everywhere else the `trufflehog2hdf` command appears as well.